src/main/java/com/dm/dmservicesandbox/domain/UserSignUpRequest.java

```diff
@@ -0,0 +1,73 @@
+package com.dm.dmservicesandbox.domain;
+
+import com.dm.dmservicesandbox.validation.PasswordMatches;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Size;
+
+
+@PasswordMatches
+@Getter
+@Setter
+public class UserSignUpRequest {
+
+    private Long userID;
+
+    private String userId;
+
+    @NotEmpty
+    private String displayName;
+
+    @NotEmpty
+    private String email;
+
+    @Size(min = 6, message = "{Size.userDto.password}")
+    private String password;
+
+    @NotEmpty
+    private String matchingPassword;
```

Comment:
Should we pass the matching password? I would suppose that we use this on front end to make sure they did no typo, but its not really something we need on the backend, do we?